caddy/Caddyfile

```diff
@@ -1,41 +1,24 @@
 {$DOMAIN}:443 {
-  # Caddy on port 443 in container to vaultwarden private instance 
-  # Use it if Caddy exposed to the net 
-  # Doc about automatic HTTPS https://caddyserver.com/docs/automatic-https
-
   log {
-	output file {$LOG_FILE} {
-        roll_size 50MiB # https://caddyserver.com/docs/caddyfile/directives/log#log
-        roll_keep 5 # https://caddyserver.com/docs/caddyfile/directives/log#log
-    }
     level INFO
+    output file {$LOG_FILE} {
+      roll_size 10MB
+      roll_keep 10
+    }
   }
 
-  # Uncomment only one of the 2 lines. Depending if you provide your own cert or request one from Let's Encrypt
-  # tls {$SSLCERTIFICATE} {$SSLKEY}
+  # Use the ACME HTTP-01 challenge to get a cert for the configured domain.
   tls {$EMAIL}
 
-  encode zstd gzip
-
-  header / {
-       # Enable HTTP Strict Transport Security (HSTS)
-       Strict-Transport-Security "max-age=31536000;"
-       # Enable cross-site filter (XSS) and tell browser to block detected attacks
-       X-XSS-Protection "1; mode=block"
-       # Disallow the site to be rendered within a frame (clickjacking protection)
-       X-Frame-Options "DENY"
-       # Prevent search engines from indexing (optional)
-       X-Robots-Tag "none"
-       # Server name remove
-       -Server
-   }
-  # The negotiation endpoint is also proxied to Rocket
-  reverse_proxy /notifications/hub/negotiate bitwarden:80
-
-  # Notifications redirected to the websockets server
-  reverse_proxy /notifications/hub bitwarden:3012
-
-  # Proxy the Root directory to Rocket
-  reverse_proxy bitwarden:80
+  # This setting may have compatibility issues with some browsers
+  # (e.g., attachment downloading on Firefox). Try disabling this
+  # if you encounter issues.
+  encode gzip
 
+  # Proxy everything Rocket
+  reverse_proxy vaultwarden:80 {
```

Comment:
Did you rename the container in your setup? In the example from this repository, it's still `bitwarden`.

Reply:
You are true. I've updated it.

Also added support for push notifications introduced in v1.29.0